static/css/components/buttonCta.less

```diff
@@ -26,6 +26,7 @@ a.cta-btn {
   }
   &--available {
     background-color: @primary-blue;
+    margin-top:4px;
```

Comment:
I'm not sure this is the right solution (there's no reason inherently that the available button should have a `margin-top`.

I agree the desired solution will be a simple change, but I think the css will probably apply to the component containing the "available" + "return" buttons (rather than on the call-to-action buttons themselves)

Reply:
@mekarpeles 
I agree that the above solution gives `margin-top` for all the `cta-btn--available` buttons which do not make much sense.
Rather as you suggested the css should be applied to the component containing the "available" + "return" buttons. But there is some problem here.
```
$if user_loan:
  <a href="$borrow_link" title="Borrow from $contributor"
     data-userid="$(user_loan['userid'])" id="read_ebook"
     class="cta-btn cta-btn--available">Read eBook</a>

    $ return_url = page.url().rsplit('/', 1)[0] + '/do_return/borrow'
    <form action="$return_url" method="post" class="waitinglist-form return-book">
      <input type="hidden" name="action" value="return" />
      <input type="submit" value="Return eBook" class="cta-btn cta-btn--available" id="return_ebook"/>
    </form>
```
So the read button and the return button are not alike. The return button is wrapped by a `<form>` tag. So wrapping both of them with a `<div>` also does not make much sense as per me.

So the solution which I can think of changing the css of only return button by it's `id`. Would that be fine?

Reply:
Good catch.., This is a layout rule not a component rule. An id selector would be bad. Can we use a sibling selector ? (~ or +) or an existing containing element with this as a child?

Reply:
I think it's somewhat reasonable for us to assume "return book" only needs padding above it if it occur after "read book".

So as @jdlrobson suggests, we can follow https://css-tricks.com/almanac/selectors/a/adjacent-sibling/

Therefore, something similar to this (untested) may work:
`.cta-btn + form input.cta-btn, .cta-btn + .cta-btn { ... }`

If two tags containing `cta-btn` are next to each other... Or a `cta-btn` is next to a form with an `<input>` that has class `cta-btn` then add e.g. `margin-top: 5px`

Reply:
I just tested the below and it seems to work on the editions page:

```
.cta-btn + form input.cta-btn, .cta-btn + .cta-btn {
    margin-top: 5px;
}
```

Reply:
@jdlrobson any thoughts on where this code may go? `static/css/base/helpers-misc`?

Reply:
Adding to static/css/components/buttonCta.less sounds fine to me. It involves cta-btn so should be kept with other cta-btn rules.

Reply:
Thanks @mekarpeles @jdlrobson!
I have updated the PR :rocket: